README.md

```diff
@@ -23,10 +32,10 @@ _"... while some certainly had a defensive purpose, the use of others is not cle
 https://en.wikipedia.org/wiki/Talaiot
 
 # Table of Contents
-1. [Setup Plugin](#setup-plugin)
+1. [Setup Plugin](#setup)
```

Comment:
Should we use only "Setup" like [in the title of this item](https://github.com/cdsap/Talaiot/pull/253/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R47)?

Reply:
done